app/controllers/course/condition/achievements_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Course::Condition::AchievementsController < Course::ConditionsController
+  before_action :achievement_condition=, only: [:edit, :update, :destroy]
```

Comment:
...okay, you should call this `load_achievement_resource` or something similar to what CanCan would name it.

Why only :edit, :update, :destroy?